javaslang/src/test/java/javaslang/control/TryTest.java

```diff
@@ -1044,6 +1051,44 @@ public void shouldReturnRecoveredValueOnFailure(){
         assertThat(Try.of(() -> {throw new IllegalStateException(FAILURE);}).recoverWith(IllegalStateException.class, success()).get()).isEqualTo(OK);
     }
 
+    @Test
+    public void shouldExecuteAndFinallyOnSuccess(){
+        MutableInteger count = new MutableInteger(0);
+
+        Try.run(() -> count.setValue(0)).andFinally(() -> count.setValue(1));
+
+        assertThat(count.getValue()).isEqualTo(1);
+    }
+
+    @Test
+    public void shouldExecuteAndFinallyTryOnSuccess(){
+        MutableInteger count = new MutableInteger(0);
+
+        Try.run(() -> count.setValue(0)).andFinallyTry(() -> count.setValue(1));
```

Comment:
\[FindBugs\] INFO: RV: Return value of Try\.andFinallyTry\(Runnable\) ignored, is this OK in javaslang\.control\.TryTest\.shouldExecuteAndFinallyTryOnSuccess\(\)